.gitignore

```
@@ -115,4 +115,6 @@ dmypy.json
 
 # Logs
 *.session
-*.session-journal
\ No newline at end of file
+*.session-journal
+logs
+database
```

```suggestion
logs/
database/
```

Maybe we can add the `/` at the back to denote directories